content/about.kr.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "개요"
+date = "2014-04-09"
+aliases = ["about-us","board"]
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+도지코인 재단은 비영리 단체이며 다음을 제공하고자 한다 :
+
+* 도지코인 가상화폐의 개발 및 변호 지원. 
+* 도지코인 트레이드마크 보호 통하여 남용과 사기 막기.
+* 도지코인의 미래를 위한 개발 로드맵과 경영을 제공.
+
+### 역사
+
+해당 재단은 2014년에 도지코인 팀원에 의하여 도지코인 프로젝트의 지원 및 경영 
+지원을 위하여 설립되었다. 2021년, 재단은 코어 개발팀원, 새로운 팀원 그리고 경험이 
+풍부한 자문원에 의하여 재활성화 되어 앞으로의 10년 동안 도지코인을 성장시킬 준비가 되었다
+
+재단의 운영 원칙을 도지코인 성명서에서 읽을 수 있다 [도지코인 성명서](/manifesto).
```

Comment:
one little suggestion: if you want you can use `/kr/manifesto` so users can directly see korean version of the manifesto